mkdocs.yml

```diff
@@ -67,103 +67,104 @@ markdown_extensions:
+          - TLS: guides/tls.md
+          - TCP routing: guides/tcp.md
+          - gRPC Routing: guides/grpc-routing.md
+          - Migrating from Ingress: guides/migrating-from-ingress.md
+          - Backend Protocol Selection: guides/backend-protocol.md
+      - API Design: guides/api-design.md
+      - CRD Management: guides/crd-management.md
+      - Implementer's Guide: guides/implementers.md
   - Reference:
-    - API Types:
-      - Gateway: api-types/gateway.md
-      - GatewayClass: api-types/gatewayclass.md
-      - GRPCRoute: api-types/grpcroute.md
-      - HTTPRoute: api-types/httproute.md
-      - Policy:
-        - BackendTLSPolicy: api-types/backendtlspolicy.md
-      - ReferenceGrant: api-types/referencegrant.md
-    - API specification: reference/spec.md
-    - Policy Attachment: reference/policy-attachment.md
+      - API Types:
+          - Gateway: api-types/gateway.md
+          - GatewayClass: api-types/gatewayclass.md
+          - GRPCRoute: api-types/grpcroute.md
+          - HTTPRoute: api-types/httproute.md
+          - Policy:
+              - BackendTLSPolicy: api-types/backendtlspolicy.md
+          - ReferenceGrant: api-types/referencegrant.md
+      - API specification: reference/spec.md
+      - Policy Attachment: reference/policy-attachment.md
   - Enhancements:
-    - Overview: geps/overview.md
-    - Provisional:
-      - geps/gep-91/index.md
-      - geps/gep-1651/index.md
-      - geps/gep-1867/index.md
-      - geps/gep-2648/index.md
-      - geps/gep-2649/index.md
-    - Implementable:
-      - geps/gep-995/index.md
-    - Experimental:
-      - geps/gep-1016/index.md
-      - geps/gep-1619/index.md
-      - geps/gep-1742/index.md
-      - geps/gep-1748/index.md
-      - geps/gep-1762/index.md
-      - geps/gep-1897/index.md
-      - geps/gep-1911/index.md
-      - geps/gep-2162/index.md
-      - geps/gep-2257/index.md
-    - Standard:
-      - geps/gep-709/index.md
-      - geps/gep-718/index.md
-      - geps/gep-724/index.md
-      - geps/gep-726/index.md
-      - geps/gep-746/index.md
-      - geps/gep-820/index.md
-      - geps/gep-851/index.md
-      - geps/gep-957/index.md
-      - geps/gep-1323/index.md
-      - geps/gep-1364/index.md
-      - geps/gep-1426/index.md
-      - geps/gep-1686/index.md
-      - geps/gep-1709/index.md
-    - Memorandum:
-      - geps/gep-713/index.md
-      - geps/gep-917/index.md
-      - geps/gep-922/index.md
-      - geps/gep-1324/index.md
-      - geps/gep-2659/index.md
-      - geps/gep-2722/index.md
-      - geps/gep-2907/index.md
-    - Declined:
-      - geps/gep-735/index.md
-      - geps/gep-1282/index.md
+      - Overview: geps/overview.md
+      - Provisional:
+          - geps/gep-91/index.md
+          - geps/gep-1651/index.md
+          - geps/gep-1867/index.md
+          - geps/gep-2648/index.md
+          - geps/gep-2649/index.md
+      - Implementable:
+          - geps/gep-995/index.md
+          - geps/gep-1731/index.md
```

Comment:
This was intended to be the only changes in this file, can revert the YAML auto-formatting is undesirable.

Reply:
Yeah, I have had this problem with VSCode's YAML support before as well. I ended up needing to edit this file in Nano.

We probably should just update it in a separate PR and tidy this up for VSCode users so it doesn't keep happening.